### PR TITLE
[Feature] Enhance Metadata layout style

### PR DESCRIPTION
**Pull request by zhangmo8**
### Purpose

MetaData breadcrumb style finish & tabs style finish & tabs internationalization completed

### CODE REVIEW

**Review by songjianet — APPROVED** [review]

**Review by labbomb — APPROVED** [review]
LGTM.

File: paimon-web-ui-new/auto-imports.d.ts [hunk 1]
```diff
@@ -5,9 +5,7 @@ regarding copyright ownership.  The ASF licenses this file
 to you under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance
 with the License.  You may obtain a copy of the License at
-
```
Comment (SteNicholas) [inline paimon-web-ui-new/auto-imports.d.ts:8]: Revert this change.

File: paimon-web-ui-new/auto-imports.d.ts [hunk 2]
```diff
@@ -5,9 +5,7 @@ regarding copyright ownership.  The ASF licenses this file
...
   http://www.apache.org/licenses/LICENSE-2.0
-
```
Comment (SteNicholas) [inline paimon-web-ui-new/auto-imports.d.ts:10]: Ditto.

File: paimon-web-ui-new/auto-imports.d.ts (hunk 1 above)
Comment (SteNicholas) [inline paimon-web-ui-new/auto-imports.d.ts:8]: Revert this change.

File: paimon-web-ui-new/auto-imports.d.ts (hunk 2 above)
Comment (SteNicholas) [inline paimon-web-ui-new/auto-imports.d.ts:10]: Ditto.

File: paimon-web-ui-new/src/locales/en/modules/metadata.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+export default {
+  table_info: 'Table',
+  option_info: 'Option',
+  schema_info: 'Schema',
+  snapshot_file: 'Snapshot',
+  manifests_file: 'Manifests',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/en/modules/metadata.ts:23]:
```suggestion
  manifest_file: 'Manifest File',
```

File: paimon-web-ui-new/src/locales/en/modules/metadata.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+export default {
+  table_info: 'Table',
+  option_info: 'Option',
+  schema_info: 'Schema',
+  snapshot_file: 'Snapshot',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/en/modules/metadata.ts:22]:
```suggestion
  snapshot_file: 'Snapshot File',
```

File: paimon-web-ui-new/src/views/metadata/components/breadcrumb/index.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import { Folder } from '@vicons/ionicons5'
+
+export default defineComponent({
+  name: 'MetaDataBreadcrumb',
+  setup() { },
+  render() {
+    return (
+      <n-breadcrumb>
+        <n-breadcrumb-item>
+          <n-icon component={Folder} /> 北京总行</n-breadcrumb-item>
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/breadcrumb/index.tsx:27]: Why use these banks?

File: paimon-web-ui-new/src/locales/zh/modules/metadata.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+export default {
+  table_info: 'Table 信息',
+  option_info: 'Option 信息',
+  schema_info: 'Schema 信息',
+  snapshot_file: 'Snapshot 文件',
+  manifests_file: 'Manifests 文件',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/zh/modules/metadata.ts:23]:
```suggestion
  manifest_file: 'Manifest 文件',
```

File: paimon-web-ui-new/src/views/metadata/components/metadata-tabs/index.tsx [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+            Option 信息
+          </n-tab-pane>
+          <n-tab-pane name='schema' tab={this.t('metadata.schema_info')}>
+            Schema 信息
+          </n-tab-pane>
+          <n-tab-pane name='snapshot' tab={this.t('metadata.snapshot_file')}>
+            Snapshot 文件
+          </n-tab-pane>
+          <n-tab-pane name='manifest' tab={this.t('metadata.manifests_file')}>
+            Manifests 文件
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/metadata-tabs/index.tsx:58]:
```suggestion
            Manifest 文件
```

**Review by SteNicholas — CHANGES_REQUESTED** [review]
@zhangmo8, thanks for the contribution. I left some comments for this pull request. PTAL.

File: paimon-web-ui-new/src/views/metadata/components/breadcrumb/index.tsx (hunk 5 above)
Comment (zhangmo8) [reply paimon-web-ui-new/src/views/metadata/components/breadcrumb/index.tsx:27]: It's just a placeholder, it will become the folder name after the api is integrated.

File: paimon-web-ui-new/src/views/metadata/components/metadata-tabs/index.tsx (hunk 7 above)
Comment (zhangmo8) [reply paimon-web-ui-new/src/views/metadata/components/metadata-tabs/index.tsx:58]: Placeholder, which will become the Manifest File content area later.

File: .gitignore [hunk 8]
```diff
@@ -71,3 +71,6 @@ coverage
 *.njsproj
 *.sln
 *.sw?
+
+# paimon-web-ui-new
+paimon-web-ui-new/auto-imports.d.ts
```
Comment (SteNicholas) [inline .gitignore:76]: New line at the end of the file.

**Review by SteNicholas — APPROVED** [review]
LGTM. Thanks for @zhangmo8 updates.
